accounting/transferwise/admin.py

```diff
@@ -34,13 +34,13 @@
 class TransferWiseBulkPaymentAdmin(UuidModelAdmin):
     """ Admin configuration for the `TransferWiseBulkPayment` model. """
 
-    list_display = ('__str__', 'csv',)
+    list_display = ('__str__', 'csv_link',)
     list_filter = (('date', DateRangeFilter),)
-    readonly_fields = ('csv',)
+    readonly_fields = ('csv_link',)
     exclude = ('csv_path',)
 
-    def csv(self, instance):
-        """The invoice's clickable PDF path."""
+    def csv_link(self, instance):
+        """The bulk payment's clickable CSV path."""
```

Comment:
@UmanShahzad Thanks for catching this :)

Reply:
:+1: